DESCRIPTION

```diff
@@ -39,7 +40,9 @@ Imports:
     XML (>= 3.99-0),
     RColorBrewer (>= 1.1-2),
     cli (>= 2.0.2),
-    docopt (>= 0.7.1)
+    docopt (>= 0.7.1),
+    doParallel (>= 1.0.17),
+    doSNOW (>= 1.0.20)
```

Comment:
Could you remove these dependencies from DESCRIPTION, and mention them on the Notebook? This still a tentative (undocumented) organization for dependencies I have been following, but basically DESCRIPTION holds dependencies that Kaiaulu API (i.e. R/*) functions depend on. Meanwhile, dependencies only a Notebook currently depends on are deferred to the user of that particular Notebook to setup. This is consistent to the philosophy of only needing to setup the third party tool of interest for the Notebook.

If too many Notebooks start relying on them, then the inconvenience is a reminder that said functionality should be transferred to within the API, rather than live on the Notebook. How well this will scale, versus try to keep up with all the tools remains to be seen. But at the very least, it has prevented me to too quickly commit to a particular library or approach, while having them still be usable :)